### PR TITLE
Redis batch

### PR DESCRIPTION
**Pull request by romach**

### CODE REVIEW

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java [hunk 1]
```diff
@@ -16,6 +18,22 @@ public static void addBitcoin( BitcoinRecord bitcoin )
     System.out.println( "RedisFiller :: Id = " + bitcoin.getId() + " count = " + ++count  );
   }
 
+  public static void addBitcoins(List<BitcoinRecord> bitcoins )
```
Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java]: avoid using static methods in prefer to DI

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java [hunk 2]
```diff
@@ -16,6 +18,22 @@ public static void addBitcoin( BitcoinRecord bitcoin )
...
+  {
+    Jedis jedis = new Jedis( RedisProperties.getInstance().getRedisContainerHost(),
+        RedisProperties.getInstance().getRedisContainerExternalPort());
+    Pipeline pipeline = jedis.pipelined();
+    bitcoins.forEach(
+        b -> {
+          pipeline.hset(RedisProperties.getInstance().getRedisHashsetName(), b.getId().toString(), b.getKey() );
```
Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java]: move constant values to constant or another variables

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java [hunk 3]
```diff
@@ -16,6 +18,22 @@ public static void addBitcoin( BitcoinRecord bitcoin )
...
+          System.out.println( "RedisFiller :: Id = " + b.getId() + " count = " + ++count  );
+        }
+    );
+    pipeline.sync();
+    jedis.close();
```
Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java]: use try-with-resources here

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java [hunk 4]
```diff
@@ -16,6 +18,22 @@ public static void addBitcoin( BitcoinRecord bitcoin )
...
+    System.out.println("Bitcoin batch: " + bitcoins.size());
```
Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java]: use logging framework

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFillerScheduledTask.java [hunk 5]
```diff
@@ -18,18 +18,19 @@
...
 
 @Component
 public class RedisFillerScheduledTask {
 
     @Value("redis-data.xml")
     private Resource redisDataFile;
 
-    private int curPos = 1;
+    private int curPos = 0;
     // TODO: add StAX
```
Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFillerScheduledTask.java]: aint it stax already?

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java (hunk 2 above)
Comment (AnatoliiStepaniuk) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java]: Do you mean properties values?

Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java]: I mean that `RedisProperties.getInstance().getRedisHashsetName()` in constant and calling it every time litters code

**Comment by schaffe** [comment]
Calling your commits like "small fixes" in considered as bad practice. Please provide more descriptive commit messages

File: redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java [hunk 6]
```diff
@@ -16,7 +18,23 @@ public static void addBitcoin( BitcoinRecord bitcoin )
                              RedisProperties.getInstance().getRedisContainerExternalPort());
     jedis.hset( RedisProperties.getInstance().getRedisHashsetName(), bitcoin.getId().toString(), bitcoin.getKey() );
     jedis.close();
-    log.info( "RedisFiller :: Id = " + bitcoin.getId() + " count = " + ++count  );
+    System.out.println( "RedisFiller :: Id = " + bitcoin.getId() + " count = " + ++count  );
```
Comment (ardzi007) [inline redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java:21]: remove System.out.println and use log.info

**Pull request merged**